Header header
time send_time
uint8[] RangeImage
uint8[] IntensityMap
uint8[] AzimuthMap
uint8[] NansRow 
uint16[] NansCol 